projects/fw_102_homework/src/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdint.h>
+#include "interrupt.h"
```

Comment:
Nitpick: there should be a blank line between the standard imports and the MS-specific imports.